src/views/textview/text-view-middle.js

```diff
@@ -109,7 +109,8 @@ export class TextViewMiddle extends LitElement {
     let positionFlag = 0;
     let parallelCounter = 0;
     let parallels = [...this.selectedParallels];
-
+    console.log('PARALLELS', parallels);
```

Comment:
Can you remove these console logs please.

Reply:
Yes sorry for this!